docs/release-v1-perf-scale-benchmarks.md

```diff
@@ -0,0 +1,71 @@
+### Benchmarks
+
+This document shares some of the performance benchmarks observed as part of the v1 release.
+It will talk about what this means for an end-user's perf and scale story.
+
+#### Background: How to interpret the numbers?
+
+KubeVirt is an extension for Kubernetes that includes a collection of custom resource definitions
+(CRDs) served by kubevirt-apiserver. These CRDs are managed by controllers. Due to the distributed
+nature of the system, understanding performance and scalability data becomes challenging without
+taking specific assumptions into account. This section aims to provide clarity on those assumptions.
+
+1. The data presented in the document is collected by `periodic-kubevirt-e2e-k8s-1.25-sig-performance`.
+1. The test creates 100 minimal VMIs, with a small pause of 100 ms between creation of 2 VMIs. The definition
+   of minimal VMIs can be found here https://github.com/kubevirt/kubevirt/blob/release-1.0/tests/performance/density.go#L273
+1. The test waits for the VMIs to go into running state and collects a bunch of metrics
+1. The collected metrics are categorized into two buckets, performance and scale
+   1. Performance Metrics: This tells users how KubeVirt stack is performing. Examples include
+      `vmiCreationToRunningSecondsP50` and `vmiCreationToRunningSecondsP95`. This helps users understand how KubeVirt 
+       performance evolved over the releases; depending on the user deployment, the numbers will vary, because a real
+       production workload could use other KubeVirt extension points like the device plugins, custom scheduler, 
+       different version of kubelet etc. These numbers are just a guidance for how the KubeVirt codebase is performing 
+       with minimal VMIs, provided all other variables(hardware, kubernetes version, cluster-size etc) remain the same.
+   1. Scale: This helps users understand the KubeVirt scaling behaviors. Examples include, PATCH-pods-count for VMI,
+      PATCH-virtualmachineinstances-count for VMI and UPDATE-virtualmachineinstances-count for VMI. These metrics are
```

Comment:
I would add code formatting to the examples names to be consistent with the list of examples in the performance metrics section. 

```suggestion
   1. Scale: This helps users understand the KubeVirt scaling behaviors. Examples include, `PATCH-pods-count` for VMI,
      `PATCH-virtualmachineinstances-count` for VMI and `UPDATE-virtualmachineinstances-count` for VMI. These metrics are
```

Reply:
updated